langwatch/src/server/prompt-config/repositories/llm-config.repository.ts

```diff
@@ -55,19 +55,28 @@ export class LlmConfigRepository {
       },
     });
 
-    return configs.map((config) => {
-      if (!config.versions[0]) {
-        throw new TRPCError({
-          code: "NOT_FOUND",
-          message: `Prompt config ${config.id} has no versions.`,
-        });
-      }
-
-      return {
-        ...config,
-        latestVersion: parseLlmConfigVersion(config.versions[0]),
-      };
-    });
+    // This is a quick and dirty way to handle the fact that some configs
+    // may have been corrupted. They will have to be fixed manually.
+    return configs
+      .map((config) => {
+        try {
+          if (!config.versions[0]) {
+            throw new TRPCError({
+              code: "NOT_FOUND",
+              message: `Prompt config ${config.id} has no versions.`,
```

Comment:
What do you think about logging an error here, but letting the request continue? Or could this only really result from a data integrity error?